lib/common_test/doc/src/ct_hooks_chapter.xml

```diff
@@ -511,6 +511,25 @@ results(State) ->
         use another level either change the <c>default</c> handler level before
         starting common_test, or use the <seemfa marker="kernel:logger#set_handler_config/3">
         <c>logger:set_handler_config/3</c></seemfa> API.</p>
+        <p>This hook supports the following options:</p>
+        <taglist>
+          <tag><c>{mode, replace}</c></tag>
+          <item>
+            <p>Replace the <c>default</c> logging handler with <c>cth_log_redirect</c>
+              instead of logging to both the default handler and this handler.
+              This effectively silences any logger output which would normally be printed
+              to standard output during test runs. To enable this mode, you can pass
+              the following options to <c>ct_run</c>:</p>
+            <p><c>-enable_builtin_hooks false -ct_hooks cth_log_redirect [{mode,add}]</c></p>
+          </item>
+          <tag><c>{mode, add}</c></tag>
+          <item>
+            <p>Add the logging handler instead of replacing the default logging
```

Comment:
Please skip part "instead of replacing the default logging" - I think this would simplify the description.
maybe 2 first sentences could be combined into one ...?

Reply:
I've changed the first two sentences to "Add cth_log_redirect to the default logging handler: Logs will be emitted to both standard output via the default handler, and into the Common Test HTML logs.". What do you think?